src/frontend/screens/Observation/index.tsx

```diff
@@ -68,15 +65,8 @@ export const ObservationScreen: NativeNavigationComponent<'Observation'> = ({
 
   const {data: fieldsData} = useManyDocs({projectId, docType: 'field'});
   const {lat, lon, originalVersionId, metadata} = observation;
-  const {data: deviceInfo} = useOwnDeviceInfo();
-  const {data: createdByDeviceId} = useDocumentCreatedBy({
-    projectId: projectId,
-    originalVersionId,
-  });
-  const isMine =
-    deviceInfo?.deviceId !== undefined &&
-    createdByDeviceId !== undefined &&
-    deviceInfo.deviceId === createdByDeviceId;
+
+  const canDelete = useCanEditOrDelete(originalVersionId);
```

Comment:
I know this is me who originally did this. But i think we should just use the hook directly inside the button component since that component has access to the `originalVersionId`. Can delete is not needed for this component